pkg/test/ginkgo/cmd_runsuite.go

```diff
@@ -313,6 +335,11 @@ func (opt *Options) Run(suite *TestSuite, junitSuiteName string) error {
 		return strings.Contains(t.name, "[sig-cli] oc adm must-gather")
 	})
 
+	// no need to count storage tests if storage disabled
+	if !storageEnabled {
+		storageTests = nil
```

Comment:
Ah, clearing this here keeps `expectedTestCount` from being inflated with subsequently-not-added storage tests.  Maybe this is where we should log the `Skipping ...`.

Reply:
Added 9e66b24935a39a9d8f6abbab925179cf4e28d4a1